reporters/stylish.js

```diff
@@ -22,8 +22,9 @@ function Stylish (options) {
     // Loop file specific error/warning messages
     data.results.forEach(function (file) {
       file.messages.forEach(function (msg) {
-        var line = colors.yellow('line ' + msg.line + ':' + msg.column) + '\t' + colors.cyan(msg.message)
-        lines.push(line)
+		var context = colors.yellow((options.showFilePath ? filepath + ':' : 'line ') + msg.line + ':' + msg.column)
```

Comment:
Oh the irony! Sorry, really didn't mean to do that. (I wasn't using my good IDE) Correcting this now.